extensions/parquet/table/src/main/java/io/deephaven/parquet/table/MappedSchema.java

```diff
@@ -26,21 +26,14 @@ static MappedSchema create(
             final TableDefinition definition,
             final RowSet rowSet,
             final Map<String, ? extends ColumnSource<?>> columnSourceMap,
-            final ParquetInstructions instructions,
-            final ColumnDefinition<?>... extraColumns) {
```

Comment:
I verified that extraColumns was never used, even in the very first commit that added it.

Reply:
Even in the enterprise codebase?

Reply:
No issues.  Everything compiles just fine in Core+